packages/sit-onyx/src/composables/useSkeletonState.ts

```diff
@@ -39,17 +39,17 @@ export type SKELETON_INJECTED = symbol; // we can't use `typeof SKELETON_INJECTE
  * ```
  */
 
-export type SkeletonInjected = symbol | boolean;
```

Comment:
Please create a follw-up ticket for making the SkeletonInjected and LocalProps generic